tests/components/sky_remote/test_remote.py

```diff
@@ -0,0 +1,29 @@
+"""Test sky_remote remote."""
+
+from unittest.mock import MagicMock
+
+import pytest
+
+from homeassistant.components.sky_remote.remote import SkyRemote
+from homeassistant.exceptions import ServiceValidationError
+
+
+def test_send_command() -> None:
+    """Test "send_command" method."""
+    mock_remote = MagicMock()
+    remote = SkyRemote(mock_remote, "test_remote")
+    remote.send_command(
+        ["sky"],
+    )
+    mock_remote.send_keys.assert_called_once_with(["sky"])
+
+
+def test_send_invalid_command() -> None:
+    """Test "send_command" method."""
+    mock_remote = MagicMock()
+    remote = SkyRemote(mock_remote, "test_remote")
+    with pytest.raises(ServiceValidationError):
+        remote.send_command(
+            ["apple"],
+        )
+    mock_remote.send_keys.assert_not_called()
```

Comment:
We should setup the integration like we do in `test_init`. Just add the config entry and tell HA to setup the integration. Because the library is mocked out (or well, that should happen then), the integration sets up and then you can execute service actions on the entity and then assert they are called correctly at the mock